lib/stripe/checkout/session.ex

```diff
@@ -173,10 +173,14 @@ defmodule Stripe.Session do
         }
 
   @type line_item :: %{
-          optional(:object) => String.t(),
-          optional(:data) => line_item_data(),
-          optional(:has_more) => boolean,
-          optional(:url) => String.t()
+          id: Stripe.id(),
+          object: String.t(),
+          amount_subtotal: non_neg_integer,
+          amount_total: non_neg_integer,
+          currency: String.t(),
+          description: String.t(),
+          price: Stripe.Price.t(),
+          quantity: non_neg_integer
```

Comment:
https://stripe.com/docs/api/checkout/sessions/object

I believe the Session Checkout session was correct.  Perhaps I'm missing something?

<img width="555" alt="Screen Shot 2022-11-05 at 9 19 50 AM" src="https://user-images.githubusercontent.com/7374640/200124409-4d987a1f-f761-4bbb-8e40-43b7a15af01a.png">